sdk/keyvault/azkeys/crypto/crypto_client_test.go

```diff
@@ -16,6 +16,20 @@ import (
 
 var ctx = context.TODO()
 
+func TestConstructor(t *testing.T) {
+	client, err := NewClient("https://fakekvurl.vault.azure.net/keys/key89075156/0b29f1d3760f4407aeb996868c9a02a7", &FakeCredential{}, nil)
+	require.NoError(t, err)
+	require.NotNil(t, client.kvClient)
+	require.Equal(t, client.keyID, "key89075156")
```

Comment:
Is there a test that covers when key ID is missing?

Reply:
There is now